.github/workflows/parallel_ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Parallel CI
+on:
+    workflow_call:
+jobs:
+    parallel-test:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: true
+            matrix:
```

Comment:
The matrix is not needed. Since the tests complete in 9 seconds, we don't really gain anything by parallelizing them

Reply:
thanks @jrhoads. i've updated the pr.